docs/javascript/mathjax.js

```diff
@@ -0,0 +1,16 @@
+window.MathJax = {
+    tex: {
+        inlineMath: [["\\(", "\\)"]],
```

Comment:
We use LaTeX?

Reply:
Yep, in the explanation for ratelimits and in some random places